src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/MultipleCreationService.java

```diff
@@ -88,6 +115,105 @@ public void bulkCreationLogic(String userToken, MultipleDetails multipleDetails,
 
     }
 
+    private void addLegalRepsFromSinglesCases(MultipleDetails multipleDetails) throws IOException {
+        MultipleData multipleData = multipleDetails.getCaseData();
+        String leadCaseId = multipleData.getLeadCaseId();
+        List<String> caseIds = multipleData.getCaseIdCollection().stream()
+            .map(o -> o.getValue().getEthosCaseReference())
+            .collect(Collectors.toList());
+
+        if (!isNullOrEmpty(leadCaseId)) {
+            caseIds.add(leadCaseId);
+        }
+
+        if (caseIds.isEmpty()) {
+            // No cases linked to this multiple means no legal reps to add
+            return;
+        }
+
+        String token = adminUserService.getAdminUserToken();
+        String multipleCaseTypeId = multipleDetails.getCaseTypeId();
+        String singleCaseTypeId = multipleCaseTypeId.replace("_Multiple", "");
```

Comment:
There is a MultiplesHelper.removeMultipleSuffix() that could potentially be used here.